pytest.ini

```diff
@@ -2,3 +2,6 @@
 markers =
     serial: tests that cannot be reliably ran with pytest multiprocessing
     timeout: used with pytest-timeout
+addopts =
+    --color yes
+    -r a
```

Comment:
I recommend making use of the strict options.
Here's a few annotated examples I have: https://github.com/ansible/pylibssh/blob/devel/pytest.ini / https://github.com/cherrypy/cheroot/blob/master/pytest.ini.
Pay special attention at `--strict-markers` / `xfail_strict` / `filterwarnings = error` and convenience settings like `--showlocals` / `--durations=10`